app/controllers/active_storage/disk_controller.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+# Serves files stored with the disk service in the same way that the cloud services do.
+# This means using expiring, signed URLs that are meant for immediate access, not permanent linking.
+# Always go through the BlobsController, or your own authenticated controller, rather than directly
+# to the service URL.
+class ActiveStorage::DiskController < ActiveStorage::BaseController
```

Comment:
can you add some README.md files to just make it easy to retrace steps of what might not be obvious if we have to revisit this in a 6 months?  I know you ended up going into internals bc we are using a react front end so whatever knowledge you have would be good to put somewhere.

Reply:
Yikes, forgot to remove these active storage files. I had just copied them over here, so i could easily poke around to see what active storage is doing. removed them as i didn't have to monkey patch anything.